src/components/plots/DoubletScoresPlot.jsx

```diff
@@ -4,31 +4,91 @@ import PropTypes from 'prop-types';
 import { Vega } from 'react-vega';
 
 import { Skeleton } from 'antd';
+
 import PlatformError from '../PlatformError';
-import generateSpec from '../../utils/plotSpecs/generateDoubletScoresSpec';
+import { generateSpec, generateData } from '../../utils/plotSpecs/generateDoubletScoresSpec';
 import loadCellMeta from '../../redux/actions/cellMeta';
+import { loadCellSets } from '../../redux/actions/cellSets';
+import { loadEmbedding } from '../../redux/actions/embedding';
+import { loadProcessingSettings } from '../../redux/actions/experimentSettings';
 
 const DoubletScoresPlot = (props) => {
   const {
-    experimentId, config, plotData, actions,
+    experimentId, config, actions, plotUuid,
   } = props;
+  const embeddingType = 'umap';
```

Comment:
Shouldn't this be taken from config?

Reply:
Turns out it is not used anymore, so I'm removing it.